demos/rforward.py

```diff
@@ -80,7 +81,7 @@ def reverse_forward_tunnel(server_port, remote_host, remote_port, transport):
         thr = threading.Thread(
             target=handler, args=(chan, remote_host, remote_port)
         )
-        thr.setDaemon(True)
+        set_daemon(thr, True)
```

Comment:
We could use simply `thr.daemon = True` with Python 2.7 and 3.

* https://docs.python.org/2/library/threading.html#threading.Thread.daemon
* https://docs.python.org/3/library/threading.html#threading.Thread.daemon

Reply:
Woops I got the description wrong, this is for py2.6 support

Reply:
The `daemon` attribute is available since Python 2.6 and works.

Reply:
I've confirmed with Python 2.6 and just confirmed now.

```
$ python2.6
Python 2.6.9 (default, Jul 29 2017, 12:47:14)
[GCC 5.4.0 20160609] on linux4
Type "help", "copyright", "credits" or "license" for more information.
>>> import threading
>>> import time
>>> t = threading.Thread(target=lambda: time.sleep(1000))
>>> t.daemon = True
>>> t.start()
>>> t.is_alive()
True
>>> #=> it is able to exit without no wait
```

Reply:
> Wait only 3.3 needs the old method names? It seems the new way is supported from 2.6+? [#1870 (comment)](https://github.com/paramiko/paramiko/pull/1870#issuecomment-1063258752)

I recommend checking documentation. The `daemon` and `notify_all` are available on Python 3.0.

* https://docs.python.org/3.0/library/threading.html#threading.Thread.daemon
* https://docs.python.org/3.0/library/threading.html#threading.Condition.notify_all